lib/WebService/MinFraud.pm

```diff
@@ -8,5 +8,77 @@ our $VERSION = '0.001001';
 
 1;
 
-# ABSTRACT: Perl API for MaxMind's minFraud web services
+# ABSTRACT: Perl API for MaxMind's minFraud Score and Insights web services
 
+__END__
+
+=pod
+
+=head1 SYNOPSIS
+
+  use 5.010;
+
+  use WebService::MinFraud::Client;
+
+  # The Client object can be re-used across several requests.
+  # Your MaxMind user_id and license_key are available at
+  # https://www.maxmind.com/en/my_license_key
+  my $client = WebService::MinFraud::Client->new(
+      user_id     => '42',
```

Comment:
The number doesn't need quotes here and user_id is actually a type of PostiveInt.  The rest of the examples don't quote the user_id.